psi4/share/psi4/fsapt/fsaptdiff.py

```diff
@@ -57,62 +57,62 @@
 
     fh = open('%s/fsapt.dat' % dirA, 'w')
     fh, sys.stdout = sys.stdout, fh
-    print '  ==> F-ISAPT: Links by Charge <==\n'
+    print('  ==> F-ISAPT: Links by Charge <==\n')
     stuffA = computeFsapt(dirA, False)
-    print '   => Full Analysis <=\n'
+    print('   => Full Analysis <=\n')
     printOrder2(stuffA['order2'], stuffA['fragkeys']) 
-    print '   => Reduced Analysis <=\n'
+    print('   => Reduced Analysis <=\n')
     printOrder2(stuffA['order2r'], stuffA['fragkeysr']) 
```

Comment:
@loriab, I'm fine with it being present or omitted.  I commented in the mistaken belief that py2.7 would be unhappy.  If you do add the extra import, `psi4/share/psi4/scripts/ixyz2database.py` also needs one.